dev/src/main/java/org/eclipse/codewind/intellij/module/TemplateTableModel.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.codewind.intellij.module;
+
+import org.eclipse.codewind.intellij.core.connection.ProjectTemplateInfo;
+import org.eclipse.codewind.intellij.core.constants.ProjectLanguage;
+import org.eclipse.codewind.intellij.core.constants.ProjectType;
+import org.eclipse.codewind.intellij.ui.messages.CodewindUIBundle;
+
+import javax.swing.table.AbstractTableModel;
+import java.util.List;
+
+public class TemplateTableModel extends AbstractTableModel {
+
+    private ProjectTemplateInfo[] templates = {};
+
+    public void update(List<ProjectTemplateInfo> templates) {
+        this.templates = templates.toArray(new ProjectTemplateInfo[0]);
+        fireTableDataChanged();
+    }
+
+    @Override
+    public int getRowCount() {
+        return templates.length;
+    }
+
+    @Override
+    public int getColumnCount() {
+        return 3;
```

Comment:
Perhaps add a comment here to say there are only 3 columns here and it is fixed?